src/test/java/seedu/guest/testutil/TypicalGuests.java

```diff
@@ -27,42 +27,42 @@ public class TypicalGuests {
     public static final Guest ALICE = new GuestBuilder().withName("Alice Pauline")
             .withEmail("alice@example.com")
             .withDateRange("13/09/22 - 15/09/22").withNumberOfGuests("1")
-            .withPhone("94351253").withIsRoomClean("yes").withBill("0")
+            .withPhone("94351253").withIsRoomClean("yes").withBill("0").withRequest("Nothing")
             .build();
     public static final Guest BENSON = new GuestBuilder().withName("Benson Meier")
             .withEmail("johnd@example.com")
             .withDateRange("30/09/22 - 02/10/22").withNumberOfGuests("2")
-            .withPhone("98765432").withIsRoomClean("yes").withBill("10")
+            .withPhone("98765432").withIsRoomClean("yes").withBill("10").withRequest("clean my room")
             .build();
     public static final Guest CARL = new GuestBuilder().withName("Carl Kurz").withPhone("95352563")
             .withEmail("heinz@example.com").withDateRange("30/12/22 - 02/01/23")
-            .withNumberOfGuests("3").withIsRoomClean("no").withBill("19.99")
+            .withNumberOfGuests("3").withIsRoomClean("no").withBill("19.99").withRequest("Room service")
             .build();
     public static final Guest DANIEL = new GuestBuilder().withName("Daniel Meier").withPhone("87652533")
             .withEmail("cornelia@example.com").withDateRange("29/05/23 - 03/06/23")
-            .withNumberOfGuests("4").withIsRoomClean("yes").withBill("50")
+            .withNumberOfGuests("4").withIsRoomClean("yes").withBill("50").withRequest("Room service")
             .build();
     public static final Guest ELLE = new GuestBuilder().withName("Elle Meyer").withPhone("9482224")
             .withEmail("werner@example.com").withDateRange("08/04/23 - 22/04/23")
-            .withNumberOfGuests("1").withIsRoomClean("no").withBill("100")
+            .withNumberOfGuests("1").withIsRoomClean("no").withBill("100").withRequest("Room service")
```

Comment:
Should we have more varied requests for typical guests?

Reply:
fixed all things